src/bindgen.rs

```diff
@@ -1,24 +1,67 @@
 use emoji;
 use error::Error;
 use progressbar::Step;
-use std::process::Command;
+use std::{path, process::Command};
 use PBAR;
 
-pub fn cargo_install_wasm_bindgen(step: &Step) -> Result<(), Error> {
+#[cfg(not(target_family = "windows"))]
+static LOCAL_BINDGEN_PATH: &str = "bin/wasm-bindgen";
```

Comment:
this is local to the crate- correct? if so we'll want to add it to the .gitignore